### ISSUE TITLE
Bug - SearchInput - advanced example toggle does not have aria-expanded by default

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11399]
**Describe the problem**
In the advanced search input example, the toggle that opens the advanced search does not have the `aria-expanded` attribute on page load. It only renders the attribute after clicking it once, at which point it updates between true and false as expected.

**How do you reproduce the problem?**
Go to the example and open dev tools, notice the button does not have `aria-expanded`. Then click the toggle a couple of times and notice the attribute gets added and updated as expected.

**Expected behavior**
The aria-expanded attribute would be present by default with a value based on its initial expanded state (false in the case of this example).

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**
